.github/workflows/codeql-analysis.yml

```diff
@@ -32,40 +32,39 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'python' ]
+        language: ["python"]
         # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby' ]
         # Learn more about CodeQL language support at https://git.io/codeql-language-support
-    permissions: read-all
```

Comment:
Is removing the permissions intentional?

Reply:
yes there is already permission defined above